### PR TITLE
🐛  Remove oldschool HTML attribrutes

### PR DESCRIPTION
**Pull request by aileen**
no issue

Uses `allowedAttributes` functionality of `Sanitize` HTML and whitelists attributes for certain tags, regarding AMP validation rules.

This PR fixes issues with inline style like `border`, `bgcolor`, `align` and so on.

### CODE REVIEW

File: core/server/apps/amp/lib/helpers/amp_content.js [hunk 1]
```diff
@@ -81,11 +158,7 @@ function ampContent() {
         // errors in video, because video will be stripped out.
         // @TODO: remove this, when Amperize support video transform
         $('video').children('source').remove();
-
-        // Vimeo iframe e. g. come with prohibited attributes
-        // @TODO: remove this, when Amperize supports HTML sanitizing
-        $('amp-iframe').removeAttr('webkitallowfullscreen');
-        $('amp-iframe').removeAttr('mozallowfullscreen');
+        $('video').children('track').remove();
```
Comment (kirrg001) [inline core/server/apps/amp/lib/helpers/amp_content.js]: `$('*').removeAttr('style');` is redundant now with the new logic or?

Comment (AileenCGN) [inline core/server/apps/amp/lib/helpers/amp_content.js]: No, because of `amp-*: ['*']` which would allow further amp-HTML tags and allow all remaining attributes. But style is one, that always has to be stripped out.

Comment (kirrg001) [inline core/server/apps/amp/lib/helpers/amp_content.js]: if i remove the line, the `amp_content_spec.js` test is still green, that's why i wondered

Comment (AileenCGN) [inline core/server/apps/amp/lib/helpers/amp_content.js]: I did write a test for that specifically. Can do it quickly. It'a mostly a fallback.

Comment (kirrg001) [inline core/server/apps/amp/lib/helpers/amp_content.js]: you can do as a separate PR, this is already green. so would like to merge now 👍

Comment (kirrg001) [inline core/server/apps/amp/lib/helpers/amp_content.js]: its not green, didn't start even haha. ok then feel free to add here 👍

Comment (AileenCGN) [inline core/server/apps/amp/lib/helpers/amp_content.js]: I was wondering already... Update comes in two minutes

Comment (AileenCGN) [inline core/server/apps/amp/lib/helpers/amp_content.js]: Bummer 😔  
`amp-*` in allowed tags didn't work, therefore, `amp-*: ['*']` in not necessary and make also `$('*').removeAttr('style')` redundant. I changed it to only allow our supported or AMPs built in tags and will push these changes.

Comment (kirrg001) [inline core/server/apps/amp/lib/helpers/amp_content.js]: supi 👍
